src/server/pegasus_server_impl.cpp

```diff
@@ -2113,6 +2139,62 @@ void pegasus_server_impl::update_server_rocksdb_statistics()
     }
 }
 
+void pegasus_server_impl::update_server_rocksdb_rate_limiter_statistics(
+    std::chrono::seconds update_rdb_stat_short_interval)
+{
+    // Update _pfc_rdb_write_limiter_rate_bytes
+    if (_s_rate_limiter) {
+        uint64_t current_total_through = _s_rate_limiter->GetTotalBytesThrough();
+        uint64_t through_rate_per_sec =
+            (current_total_through - _rocksdb_limiter_last_total_through) /
+            update_rdb_stat_short_interval.count();
```

Comment:
Use config_flag validator to prevent update_rdb_stat_short_interval not equal to 0.